### PR TITLE
Add nodejs and npm

### PR DESCRIPTION
**Pull request by rogiervanhetschip**
Add NodeSource PPA, and install nodejs and npm

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,6 +12,8 @@ RUN mkdir -p /usr/share/man/man1 \
     dpkg \ 
     zip \
     default-jre \
+    && curl -fsSL https://deb.nodesource.com/setup_16.x | bash - \
```
Comment (Mastermindzh) [inline Dockerfile:15]: Any issues with v18?

**Review by Mastermindzh — CHANGES_REQUESTED** [review]
You're missing some stuff:

- Changelog
- README update
- Version numbering

Should be easy after that though 😄

File: Dockerfile (hunk 1 above)
Comment (rogiervanhetschip) [reply Dockerfile:15]: I used the LTS support, which is currently v16, to get a more stable version. Do you want to me switch to v18?

Comment (rogiervanhetschip) [reply Dockerfile:15]: No update to README needed, will fix the others.

**Review by Mastermindzh — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (Mastermindzh) [reply Dockerfile:15]: No, I thought 18 was TLS by now :)

**Pull request merged**